.eslintrc

```diff
@@ -49,32 +58,60 @@
         "no-unused-expressions": 2,
         "no-with": 2,
         "radix": 2,
+        "vars-on-top": 1,
```

Comment:
One of the more invasive changes. I think it's important to define all variables in one place to reduce confusing code and ease refactoring.